### DISCUSSION TITLE
5기 4주차 회고

### DISCUSSION THREAD

**Discussion by imakerjun** [next-step/blog#270] [회고]
```
# 📅 5월 4일~5월 10일의 회고
## 📝 블로그 글 소개

## 💡 지난 1달간 인사이트 있었던 일

## 🤔 지난 1달간 아쉬움이 남은 일

## 🪜 내일부터 다르게 하고 싶은 일(내일 당장 실천할 수 있는 액션 플랜과 함께)
```

**Comment by lns13301** [comment]
# 📅 5월 4일~5월 10일의 회고
## 📝 블로그 글 소개
지난 모의 면접을 보고 느낀점을 간단하게 작성했습니다.

## 💡 지난 1달간 인사이트 있었던 일
블로그에 글을 작성하거나 필요한 내용을 정리하는 양이 눈에 띄게 늘었습니다.

## 🤔 지난 1달간 아쉬움이 남은 일
게을러지는 행동이 많았었던 점

## 🪜 내일부터 다르게 하고 싶은 일(내일 당장 실천할 수 있는 액션 플랜과 함께)
우선 내일까지 진행중인 미션을 완성하고 제출하기

**Comment by catsbi** [comment]
# 📅 5월 4일~5월 10일의 회고
## 📝 블로그 글 소개
지난 한 주동안(5월 2주차) 있었던 일에대한 회고와 제가 느낀 부분들 가령 예를들어 멘탈관리에 대해 생각한 부분에 대해서 글을 작성해보았습니다. 

## 💡 지난 1달간 인사이트 있었던 일
가장 많은 시간을 쓴 인강/실무/공부가 아니라 주말에 잠깐 했던 스터디모임에서 인사이트를 얻었습니다. 
제가 요근래 약 4년정도 사람들과 트러블이 생겼던 거의 유일한 부분이 공부에 대한 것인데, 학교를 다닐적에 저는 다른 친구들에 비교해서 항상 노력도 덜 하고 놀기만했기에, 나보다도 공부를 안하면 진짜 심각한거라는 생각때문에 학원에서 팀프로젝트를 할때도, 회사에서도 나보다 공부를 적게하거나 일찍 가는 사람이 있으면 스트레스를 받아서 표정관리도 못하는 편이라서 그게 불화의 씨앗이 되기 말녀이였는데, 이번 스터디에서 얘기를 나누고 다양한 사람들의 시각을 듣다보니, 그동안 꾸준히 들었던 것들이 쌓이고 쌓인상태에서하나의 키가되서 
최근에서야 각자에겐 각자의 사정이 있고 내가 상대방을 가늠할 자격은 없다고 생각하게 되었스ㅡㅂ니다.

## 🤔 지난 1달간 아쉬움이 남은 일
3끼를 모두 매일 새벽 배송해주는 식단관리 프로그램을 비싼돈내고 시작했는데, 저녁에 샐러드 먹기싫어서 배달음식 먹은적이 꽤-많습니다. 아주많이 후회합니다. 냉장고에 차있는 샐러드를 빨리 처리해야 합니다.

## 🪜 내일부터 다르게 하고 싶은 일(내일 당장 실천할 수 있는 액션 플랜과 함께)
- 배달음식을 안먹어야합니다.
- 회사에서 좀 더 회사업무에 집중해야 합니다. 개인공부에 빠져 회사업무를 등한시하는 것 같습니다.

**Reply by imakerjun** [reply]
배달음식도 건강한게 많더라구요 😭

**Comment by imakerjun** [comment]
# 📅 5월 4일~5월 10일의 회고
## 📝 블로그 글 소개
몰타에서 느꼈던 섬세한 행복한 감정과 그 시선을 다시 적어봤습니다. 내 안에 있는 나의 특징, 보고자 하는 시선들을 다시 떠올려볼 수 있었네요 ㅎㅎ

## 💡 지난 1달간 인사이트 있었던 일
- 운동의 재미가 붙기 시작하다. 이젠 헬스장 가는게 더 즐거워짐 (신체의 피드백이 더 빠르게 옴)
- 문제를 작게 쪼개고, 작은 단위로 행동해서 시작하고 빠르게 피드백 받기 (이것이 TDD이며 애자일)
> TDD란 프로그래밍 의사결정과 피드백 사이의 간극을 의식하고 이를 제어하는 기술이다
- 실시간 코드리뷰 방식으로 개선한 후 얻은 인사이트가 굉장히 큼
- 바로 피드백을 주고받을 수 있는 독서 스터디에서 영감을 크게 얻음

## 🤔 지난 1달간 아쉬움이 남은 일
- 글을 매번 급하게 월요일에 올리니 글에 실수가 잦다.... 월요일 회고하고 나면 다음날 올리는 습관을 들여야할 것 같다.
- 업무적으로 개인적으로 새로운 시도가 많았다는 핑계로 블로그 플랫폼을 충분히 개선하지 못한 점

## 🪜 내일부터 다르게 하고 싶은 일(내일 당장 실천할 수 있는 액션 플랜과 함께)
1개의 PR에서 여러개의 링크를 달 수 있게 템플릿을 테이블 형태로 만들기

### 예시
|제목|설명|카테고리|링크|
|---|---|---|---|
|제목1|이것은|여행기|[링크]()|
|제목2|설명이다|여행기|[링크]()|

**Comment by loop-study** [comment]
# 📅 5월 4일~5월 10일의 회고
## 📝 블로그 글 소개
- 5월 1주차 회고를 작성
- 소프트웨어 장인을 챕터마다 리뷰
## 💡 지난 1달간 인사이트 있었던 일
- TDD, Clean Code With Java 를 통해 객체지향이 무엇인지 깨달음
- 객체지향에 관련 책을 읽고 객체지향에 대해 공부를 하다보니 이전까지 사용만하던 스프링을 제대로 이해하기 시작함
- 소프트웨어 장인 책을 통해 올바른 개발자라면 가져야 할 마음가짐과 태도를 하나씩 알게됨
## 🤔 지난 1달간 아쉬움이 남은 일
- 공부 시간을 좀 더 제대로 활용했으면 좋았을텐데 아쉬움이 남음.
- 책을 읽거나 공부를 하면 블로그에 남기고 싶은데 작성하는데도 시간이 걸려 글을 남기지 못한 게 있어서 아쉬움.
## 🪜 내일부터 다르게 하고 싶은 일(내일 당장 실천할 수 있는 액션 플랜과 함께)
- 아침일찍 일어나서 아침 운동가기 (밤 12시 전에 잠자기, 스마트폰 멀리하기!!)
- 스톱워치를 이용해서 공부시간을 좀 더 제대로 관리하기

**Comment by humblEgo** [comment]
# 📅 5월 4일~5월 10일의 회고
## 📝 블로그 글 소개
지난 주에 있었던 일에 대해서 주간회고를 작성해봤습니다.

## 💡 지난 1달간 인사이트 있었던 일
주변 멘토님들을 만나서 얘기를 나눠봤는데, 같은 주제에 대해 의견이 다른 것을 느끼며, 결국 실버 불렛이 없다는 것을 체감했습니다.

## 🤔 지난 1달간 아쉬움이 남은 일
코딩테스트 준비 등에 꾸준히 시간을 쓰지 못한 것이 아쉽습니다.
한편 블로그 글도 매주 내 생각이 들어간 양질의 글을 쓰고 싶었는데, 점점 시간이 부족하단 이유로 날림글을 양산한게 아쉽네요.

## 🪜 내일부터 다르게 하고 싶은 일(내일 당장 실천할 수 있는 액션 플랜과 함께)
오전 9시부터 10시 사이 1시간은 코딩테스트 준비에 시간을 쓰겠습니다.

**Comment by ddongule** [comment]
# 📅 5월 4일~5월 10일의 회고
## 📝 블로그 글 소개
Mandalart라는 개인 프로젝트를 만들며 고생했던 Input과의 사투를 글로 풀어내보았습니다! 

## 💡 지난 1달간 인사이트 있었던 일
프로젝트를 만들고 공유해보니.. 가슴이 웅장해졌어요.
React라는 새로운 프레임워크를 배워나가면서 이해가 잘 되지 않거나 어려운 부분이 있었는데, 
어느정도 그런 부분도 해소가 된 것 같기도 하고, 또 더 재미가 붙어서 추진력이 생긴 것 같아요!

## 🤔 지난 1달간 아쉬움이 남은 일
지금 미션이 굉장히 많이 밀려있씁니다. 😭 
한 번 밀리게되니 점점 쌓여가고 있습니다..
이 레거씨(?)들을 어서 처리해야합니다.. 일정을 지키자!!! 나자신!! 정신차리자!!
헐 블랙커피 스터디도 원래 열심히 하려고 했는데 Lazy해져서 잘 못들어와서 아쉽습니다. ㅠㅠㅠㅠ

## 🪜 내일부터 다르게 하고 싶은 일(내일 당장 실천할 수 있는 액션 플랜과 함께)
당장 미션들을 하나씩 차근차근 Complete하겠습니다!!!!!!

**Comment by yujo11** [comment]
# 📅 5월 4일~5월 10일의 회고
## 📝 블로그 글 소개

- [Vanilla JS 프로젝트 세팅하기](https://yujo11.github.io/javascript/Vanilla%20JS%20%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8%20%EC%84%B8%ED%8C%85%ED%95%98%EA%B8%B0/)

제가 자주 사용하는 Vanilla JS 프로젝트 세팅 방법을 정리했습니다.

부연설명까지 하나하나 달고 싶었는데 아직 달지 못 했네요ㅠㅠ

시간 나는대로 내용 보충해보도록 하겠습니다!

## 💡 지난 1달간 인사이트 있었던 일

1. 번아웃
   - 번아웃이 한번 왔었습니다.
   - 번아웃을 통해 제가 평소 스트레스 관리, 할 일 관리에 소흘했다는 것을 알게 됐습니다.
2. 페어 프로그래밍
   - 지난 한 달간 여러 명의 페어들과 만났는데 각각의 배울점이 있는 훌륭한 페어들이었습니다.
   - 페어들 덕분에 많은 인사이트를 얻으며 성장할 수 있었습니다.

## 🤔 지난 1달간 아쉬움이 남은 일

1. 블랙커피 스터디
   - 지금 하고 있는 블로그 스터디 외에도 프론트엔드 스터디 레벨2를 신청했습니다.
   - 그러나 시간에 쫓겨 블로그 스터디와 프론트엔드 스터디에 적극적으로 참여하지 못한 점이 아쉽습니다.

## 🪜 내일부터 다르게 하고 싶은 일(내일 당장 실천할 수 있는 액션 플랜과 함께)

1.  일찍 일어나기
  - 목표 : 06시
  - 기상 시간을 일정하게 지켜 오전 시간에 운동과 독서하기
